shell/plugins/steve/subscribe.js

```diff
@@ -512,7 +546,7 @@ export const actions = {
         }, { root: true });
       } else {
         // if the error is not a connect error or disconnect error, the socket never worked: log whether the current browser is safari
-        console.error(`WebSocket Connection Error [${ getters.storeName }]`, e.detail); // eslint-disable-line no-console
```

Comment:
there's probably better ways we can log it rather than to the console, in this case, it was an unfortunate casualty of me findReplacing after getting tired of having to hunt for my own console statements in the sea of other console statements we make. I can reverse it though.